mnefun/_mnefun.py

```diff
@@ -2492,34 +2503,67 @@ def _head_pos_annot(p, raw_fname, prefix='  '):
     """Locate head position estimation file and do annotations."""
```

Comment:
I can't tell, will this handle split files?

Reply:
I hope so, it is what we use for `maxwell_filter` movement compensation IIRC but I'll have to check

Reply:
Looking at the code, `run_sss_positions` looks for extra files like `-1.fif` etc., runs head pos estimation on them all, loads the results, and concatenates them. So just like loading `_raw.fif` in MNE automatically loads the `-1.fif` files as necessary, our `.pos` files should have all positions in them.